router/users.js

```diff
@@ -33,14 +36,45 @@ router.post("/register", async (req, res) => {
       profileInfo
     );
 
+    
+    res.status(201);
+    res.json(user);
+    
+  } catch (error) {
+    console.log(error);
+    res.status(401).send({error:error.message});
+  }
+});
+
+router.post('/login',trimReqBody, async (req,res) => {
+  
+
+  if (Object.values(req.body).some((f) => f == "")) {
+    throw new Error("All fields are required");
+  }
+  const {email, password} = req.body
+  try {
+
+    const user = await login(email, password);
     res.status(201);
     res.json(user);
+    
   } catch (error) {
     console.log(error);
-    res.status(500).send(error);
+    res.status(401).send({error:error.message})
```

Comment:
I guess you could do this instead: `send(error)`  and send the whole error object. However I'm not sure what is inside of it and in the case that we want only the error message you could do this:
```
 } catch ({message}) {
    res.status(401).send({error: message})
```

Reply:
{
    "error": "Email doesnt exist or password is incorrect!"
}
that is the response answer res.status(401).send({error:error.message}), for incorrect login credentials. Should we change it?